.travis.yml

```diff
@@ -18,6 +18,11 @@ script:
 
 # note before_deploy will run before each deploy provider
 before_deploy:
+  - rm -rf $TRAVIS_REPO_SLUG
```

Comment:
I don't believe this is the right place for this change. This will be executed after the script phase. I think we need to do this before the `script`